flash.h

```diff
@@ -175,10 +175,20 @@ struct flashchip {
 	uint32_t manufacture_id;
 	uint32_t model_id;
 
+	/* only page size for NOR; others for NAND */
 	/* Total chip size in kilobytes */
 	unsigned int total_size;
+	/* Chip unit size in bytes */
+	unsigned int unit_size;
+	/* Chip block size in bytes */
+	unsigned int block_size;
 	/* Chip page size in bytes */
 	unsigned int page_size;
+	/* Chip partial page size in bytes */
+	unsigned int partial_page_size;
+	/* Data bytes per spare size */
+	unsigned int data_per_spare;
+
```

Comment:
Apart from verbose debug information, I don't see a good reason to add these. They aren't used for anything but calculations local to a function.